aiida_mlip/calculations/geomopt.py

```diff
@@ -82,6 +82,13 @@ def define(cls, spec: CalcJobProcessSpec) -> None:
             help="All other keyword arguments to pass to geometry optimizer",
         )
 
+        spec.input(
+            "pressure",
+            valid_type=Float,
+            required=False,
+            help="External pressure (float)",
```

Comment:
I don't think we need to say it's a float, since that's labelled by the `valid_type` already, but something closer to the CLI help (in GPa) is probably useful.